cypress/integration/homepage-spec.js

```diff
@@ -28,8 +28,7 @@ describe('Test home page', () => {
   it('Test empty inputs', () => {
     cy.get('button[type="submit"]').click();
     cy.get('.user-form__alert').then((alerts) => {
-      const emailAlert = alerts[0];
-      const passwordAlert = alerts[1];
+      const { emailAlert, passwordAlert } = alerts;
```

Comment:
if `alerts` is still an array, it should be like this: `const [emailAlert, passwordAlert] = alerts;`